test/testdrive/mzcompose.yml

```diff
@@ -24,6 +24,36 @@ mzworkflows:
         service: testdrive-svc
         command: --aws-endpoint=http://localstack:4566 ${TD_TEST:-*.td esoteric/*.td}
 
+  # Runs the tests that interact with Kafka against Redpanda.
+  #
+  # Specify the `TD_TEST` environment variable to select a specific test to run.
+  # Otherwise, this workflow runs against the test files in this directory that
+  # use a Kafka action but do *not* use `kafka-add-partition`, as Redpanda does
+  # not yet support dynamically adding partitions to a topic.
+  testdrive-redpanda:
+    steps:
+      - step: start-services
+        services: [redpanda, schema-registry, materialized]
+      - step: wait-for-tcp
+        host: redpanda
+        port: 9092
+        timeout_secs: 120
+      - step: wait-for-tcp
+        host: schema-registry
+        port: 8081
+      - step: wait-for-mz
+      - step: run
+        service: testdrive-svc
+        entrypoint: /bin/bash
+        command:
+        - -c
+        - >-
+          testdrive
+          --kafka-addr=redpanda:9092
+          --schema-registry-url=http://schema-registry:8081
+          --materialized-url=postgres://materialize@materialized:6875
+          ${TD_TEST:-$(grep -lF '$ kafka' $(grep -LF '$ kafka-add-partitions' *.td))}
```

Comment:
We have a less crazy bashism in the antithesis driver container in order to exclude certain tests there. At some point, it may be better to maintain a list of tests to skip or include for each mzcompose workflow as files in the directory.

Reply:
@philip-stoev - PR to remove the *last* known work around will land here https://github.com/vectorizedio/redpanda/pull/2025 wich is exciting.